src/libraries/Microsoft.Bcl.AsyncInterfaces/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.IAsyncDisposable</Target>
+    <Left>lib/netstandard2.0/Microsoft.Bcl.AsyncInterfaces.dll</Left>
```

Comment:
Do you know why are we hitting these? Why did `lib/netstandard2.1` asset removed all these types?

Reply:
netstandard2.1 is just a facade: https://github.com/dotnet/runtime/blob/0ea6d0c8e45646cc2e359c32aae970ec4baa2c5b/src/libraries/Microsoft.Bcl.AsyncInterfaces/src/Microsoft.Bcl.AsyncInterfaces.csproj#L6.